File: Python/bir işlem/entity/player.py
class Player:
    """NPC (Non-Player Character) verilerinin tutulduğu varlık sınıfı
    @category: Entity Class"""

    def __init__(self) -> None:
        """NPC (Non-Player Character) verilerinin tutulduğu varlık sınıfı
    @category: Entity Class"""
        self._name = ""
        self._score = 0.0
        self._process = ""

    @property
    def name(self) -> str:
        """NPC (Non-Player Character) 'in isim verisini geri döndürür

        Returns:
            name (str): NPC ismi"""
        return self._name

    @name.setter
    def name(self, value: str) -> None:
        """NPC (Non-Player Character) 'in isim verisini ayarlar

        Args:
            value (str): NPC ismi"""
        self._name = value

    @property
    def score(self) -> float:
        """NPC (Non-Player Character) 'in puan verisini geri döndürür

        Returns:
            score (float): NPC puanı"""
        return self._score

    @score.setter
    def score(self, value: float) -> None:
        """NPC (Non-Player Character) 'in puan verisini ayarlar

        Args:
            value (float): NPC puanı"""
        self._score = value

    @property
    def process(self) -> str:
        """NPC (Non-Player Character) 'in yaptığı işlem verilerini geri döndürür.

        Returns:
            process (str): NPC işlemleri"""
        return self._process

    @process.setter
    def process(self, value: str) -> None:
        """NPC (Non-Player Character) 'in yaptığı işlem verisini ayarlar.

        Args:
            value (str): NPC işlemi"""
        self._process = value

File: Java/bir işlem/entity/Player.java
package entity;

/**
 * NPC (Non-Player Character) varlık sınıfı.
 * 
 * @apiNote NPC bilgilerini tutar
 * @category Varlık Sınıfı
 */
public class Player {
    private String name;
    private float score;
    private String process;

    /**
     * Player varlık sınıfını parametresiz oluşturur
     */
    public Player() {
    }

    /**
     * Player varlık sınıfını parametrelerle oluşturur
     * 
     * @param name    -> NPC ismi (String)
     * @param score   -> NPC puanı (float)
     * @param process -> NPC işlem bilgisi (String)
     */
    public Player(String name, float score, String process) {
        this.name = name;
        this.score = score;
        this.process = process;
    }

    /**
     * NPC ismini verir
     * 
     * @return name -> NPC ismi (String)
     */
    public String getName() {
        return name;
    }

    /**
     * NPC ismini ayarlar
     * 
     * @param name -> NPC ismi (String)
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * NPC puanını verir
     * 
     * @return score -> NPC puanı (float)
     */
    public float getScore() {
        return score;
    }

    /**
     * NPC puanını ayarlar
     * 
     * @param score -> NPC puanı (float)
     */
    public void setScore(float score) {
        this.score = score;
    }

    /**
     * NPC'nin yaptığı işlemleri verir
     * 
     * @return process -> NPC işlemleri (String)
     */
    public String getProcess() {
        return process;
    }

    /**
     * NPC'nin yaptığı işlemleri ayarlar
     * 
     * @param process -> NPC işlemşeri (String)
     */
    public void setProcess(String process) {
        this.process = process;
    }

}
